src/schema/generic.json

```diff
@@ -741,6 +746,11 @@
             "category": "client",
             "description": "Invalid url provided"
         },
+        {
+            "code": "DomainUnexpected",
```

Comment:
DomainUnexpected sounds like our protocol domain to me 🤔 by the description, it can be replaced to `UrlInvalid `?

Reply:
True. The URL is valid, however, we ended up on a different .... website :D 
How does `WebsiteUnexpected` or `HostnameUnexpected` sound?

Reply:
Ah I see.. my intention of `UrlInvalid` was that purpose, but now I can see it's not clear :( 
`WebsiteUnexpected ` or `UnexpectedUrl` sounds better to me but I don't mind you to choose one.